pkg/crc/config/viper_config.go

```diff
@@ -0,0 +1,46 @@
+package config
+
+import (
+	"fmt"
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/spf13/viper"
+	"os"
+)
+
+var (
+	GlobalViper *viper.Viper
+	ViperConfig map[string]interface{}
+)
+
+// GetViperBool returns the value of a boolean config key
+func GetViperBool(key string) bool {
```

Comment:
Ah, this is in pkg... so in that case do not use the name `Viper` but just `GetBool`...
the same for `ViperConfig`...

in this level we should have abstracted most of it away.